### ISSUE TITLE
[BUG] New (Suggested) threads are sorted by Hot

### ISSUE THREAD

**Issue by ronakg** [laurencedawson/sync-for-reddit#1444]
**Describe the bug**  
Even when a thread's Suggested sorting is set to New, sync sorts the thread by Hot.

**Steps to Reproduce**  
Visit a thread with Suggested order set to New, typically sports game day thread like this - https://www.reddit.com/r/nba/comments/vbq68t/_/

**Expected behavior**  
Thread is correctly sorted by New

**Screenshots**  

**Smartphone (please complete the following information):**

    Sync version: v22.06.10-10:35    
    Sync flavor: free    
    Removed ads: true    
    
    Ultra user: false    
    View type: Slides    
    Push enabled: false    
    
    Device: raven    
    Model: Google Pixel 6 Pro    
    Android: 12

**Additional context**

**Comment by laurencedawson** [comment]
Can't reproduce. Setting as new looks new to me.

Suggested new might be returning something wrong but thats on reddits side.

**Comment by ronakg** [comment]
> Can't reproduce. Setting as new looks new to me.

Yes, this bug is about `New (Suggested)` sort, not about setting the sort manually to `New`. This is used by all major sports subs for game-day threads - NFL, NBA, Soccer, Cricket, etc. So, not getting the comments in the correct order when you open the thread is really really annoying.

> 
> Suggested new might be returning something wrong but thats on reddits side.

Is it possible to handle this on the app side? If Reddit is returning `New (Suggested)` as the sort then sort the thread by `New`?

**Comment by laurencedawson** [comment]
Sync only sees what the suggested sort is after Reddit returns the
comments. Hmm
